### PR TITLE
Create Library Form for shelf (see Issue #100)

### LINKED ISSUE #148: After renaming a shelf the list is not sorted
When a shelf is renamed, the renamed shelf is not sorted correctly.

### PR DESCRIPTION
**Pull request by claudia3**
**Description**
Closes #148
Replace Dialog rename/add shelf for a Fragment for consistency. 

**Affects**
Library Fragment, renaming and adding a shelf.

**Notes for Reviewer**
Please review & comment/approve.

### CODE REVIEW

File: BibBuddy/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -44,14 +44,16 @@
...
     <string name="input_shelf_name">Regalnamen eingeben</string>
     <string name="invalid_name">Bitte gib einen Namen für das Regal an.</string>
     <string name="name_exists">Sorry, den Namen gibt\'s schon!</string>
     <string name="shelf_added">Ok, wir haben das Regal hinzugefügt.</string>
     <string name="add_shelf_tooltip">Ein neues Regal anlegen.</string>
 
     <!-- Library | Rename Shelf -->
+    <string name="shelf">Regal</string>
+    <string name="rename_small">umbenennen</string>
     <string name="rename_shelf">Regal umbenennen</string>
```
Comment (SarahKurek) [inline BibBuddy/app/src/main/res/values/strings.xml:57]: redundant

File: BibBuddy/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -44,14 +44,16 @@
     <!-- Library -->
     <!-- Library | Add Shelf -->
     <string name="add_shelf">Hinzufügen</string>
-    <string name="add_new_shelf">Neues Regal</string>
+    <string name="add_new_shelf">Regal hinzufügen</string>
     <string name="input_shelf_name">Regalnamen eingeben</string>
```
Comment (SarahKurek) [inline BibBuddy/app/src/main/res/values/strings.xml:48]:
```suggestion
    <string name="input_shelf_name">Regalname</string>
```

**Review by SarahKurek — CHANGES_REQUESTED** [review]
With the libraryForm, the header and the headline are the same. I would suggest only writing "Regalname" as heading, just like in the other form. Pleas check also the other comments.
While testing, no errors occurred.

File: BibBuddy/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (claudia3) [reply BibBuddy/app/src/main/res/values/strings.xml:57]: It's not redundant because between those words the old shelf name is used.

**Comment by claudia3** [comment]
> With the libraryForm, the header and the headline are the same. I would suggest only writing "Regalname" as heading, just like in the other form. Pleas check also the other comments.
> While testing, no errors occurred.

Should I also remove the old shelf name from it?

**Review by SarahKurek — APPROVED** [review]
Now it fits and can be merged.

**Pull request merged**